service/controller/resource/masters/create_manual_intervention_needed.go

```diff
@@ -0,0 +1,13 @@
+package masters
+
+import (
+	"context"
+
+	"github.com/giantswarm/azure-operator/service/controller/internal/state"
+)
+
+// This transition function aims at detecting if the master VMSS needs to be migrated from CoreOS to flatcar.
```

Comment:
I believe this comment is copy&paste? Zap? :slightly_smiling_face: 
```suggestion
```

Reply:
removed